pkgs/desktops/deepin/core/dde-kwin/default.nix

```diff
@@ -0,0 +1,96 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, extra-cmake-modules
+, deepin-gettext-tools
+, wrapQtAppsHook
+, makeWrapper
+, dtkcore
+, qtbase
+, qtx11extras
+, gsettings-qt
+, xorg
+, libepoxy
+, deepin-kwin
+, kdecoration
+, kconfig
+, kwayland
+, kwindowsystem
+, kglobalaccel
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dde-kwin";
+  version = "5.6.5";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = "b5c00527b86f773595c786c8015d60f8be3a681b";
+    sha256 = "sha256-qXN9AwjLnqO5BpnrX5PaSCKZ6ff874r08ubCMM272tA=";
+  };
+
+  /*
+    This is the final version of dde-kwin, upstream has been archived.
+    We should remove this package when deepin-kwin release a new version.
+    We need revert Dtk::Core target in postPatch,
+    since the new dtkcore will break other deepin applications.
```

Comment:
dde-kwin is not deepin-kwin，but cmake break some deepin applications，maybe it can be solved